core/src/main/java/org/elasticsearch/index/query/PrefixQueryBuilder.java

```diff
@@ -75,7 +75,7 @@ public PrefixQueryBuilder queryName(String queryName) {
     @Override
     public void doXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject(PrefixQueryParser.NAME);
-        if (boost == -1 && rewrite == null && queryName != null) {
+        if (boost == -1 && rewrite == null && queryName == null) {
```

Comment:
If you removed rendering the short version in FuzzyQ.B. and RegExQB, why not also here and in WildCardQB?